plus/app/api/endpoints/auth.py

```diff
@@ -0,0 +1,29 @@
+import os
+from typing import Any, List
+
+from sqlalchemy.orm import Session
+from fastapi import APIRouter, Depends, HTTPException
+
+from app import crud, schemas
+from app.api import depends
+from app.core import OAuth2
+
+
+router = APIRouter()
+
+nctu_oauth = OAuth2.OAuth2(client_id=os.getenv('NCTU_CLIENT_ID'),
```

Comment:
Move `nctu_oauth` variable to `/app/core/Oauth2` and import this variable here instead. Since `nctu_oauth` should be a core object.